sql/core/src/main/java/org/apache/spark/sql/execution/ArrowColumnVector.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.vectorized;
+package org.apache.spark.sql.execution;
```

Comment:
Can we put `ArrowColumnVector` into `org.apache.spark.sql.vectorized` like `ColumnVector`?

Reply:
Most `import` changes seems to be due to this relocation from `vectorized` to `execution`.

Reply:
I think this depends on whether we want to also reorganize in this commit, or just move from SQL to Catalyst. @mccheah suggested reorganizing at the same time. I don't have a strong opinion on whether we do it at the same time or separately, but if we aren't reorganizing the v2 classes, then I agree that this should not move packages.